### PR TITLE
Made some updates to the Export PDF button

### PR DESCRIPTION
**Pull request by mikeyh-aexp**
Previously, the Download PDF button had hardcoded headers. Now the PDF will display the headers that correspond to the page you are searching on

### CODE REVIEW

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ export default class SearchResult extends React.Component {
     }
 
     renderResultBody(result) {
+//         console.log(Object.keys(this.props.results[0]));
```
Comment (henryleduc) [inline AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx]: We don't want any commented code on the master repo

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx [hunk 2]
```diff
@@ -56,16 +57,20 @@ export default class SearchResult extends React.Component {
...
-                : <div className="result-heading"><h3>{`${results.length} results`}</h3><button onClick={this.jsPdfGenerator}>Download PDF</button></div>)
+                ? <div className="result-heading"><h3>{`${results.length} result`}</h3><button class="btn btn-default" onClick={this.jsPdfGenerator}>Download PDF</button></div>
+                : <div className="result-heading"><h3>{`${results.length} results`}</h3><button class="btn btn-default" onClick={this.jsPdfGenerator}>Download PDF</button></div>)
             : '';
     }
 
     jsPdfGenerator() {
        const doc = new jsPDF();
-       const col = ["Region ID","Report ID","Status","Report Time","Report Body","Agent ID"];
+       let col = Object.keys(this.props.results[0]);
```
Comment (henryleduc) [inline AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx]: Can we make this a const does it need to be changed

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx [hunk 3]
```diff
@@ -56,16 +57,20 @@ export default class SearchResult extends React.Component {
...
        const rows = [];
-       this.props.results.map(element => rows.push([element.regionId, element.reportId, element.status, element.reportTime, element.reportBody, element.agentId]));
+       for( let i = 0; i < this.props.results.length; i++ ) {
+            let row = [];
+            Object.values(this.props.results[i]).map( elem => row.push(elem) );
+            rows.push(row);
+       }
        doc.autoTable(col, rows, { startY: 10 });
        doc.save('Results.pdf');
```
Comment (henryleduc) [inline AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx]: is results a useful name?

Comment (henryleduc) [inline AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx]: Where does this save our results.pdf also if we aren't hosting the serve how can we access this?

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx (hunk 2 above)
Comment (mikeyh-aexp) [reply AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx]: Changed back to const

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx (hunk 3 above)
Comment (mikeyh-aexp) [reply AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx]: I think 'Results' is a suitable name? What other suggestions do you H&M?

Comment (henryleduc) [reply AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx]: Maybe reports and a date and time

**Pull request merged**